app.stache

```diff
@@ -196,9 +196,9 @@
           <a on:click="mergeTool = false" class="btn btn-default" style="position: absolute; right: 2rem; margin-top: 3.5rem;">Exit without saving</a>
           <merge-tool vm:targets:from="interviews" vm:sources:from="interviews" reloadInterviews:from="reloadInterviews" open:bind="mergeTool" />
         {{/if}}
-        <interviews-toolbar interviews:from="interviews"></interviews-toolbar>
+        <interviews-toolbar interviews:bind="interviews"></interviews-toolbar>
```

Comment:
this used to be `bind` in the past, and i can't recall if the change to `from/to` was solving something a problem or warning, or if i was just 'optimizing'. worth keeping an eye on is all.

Reply:
It used to be "to", I changed it to "from" at the start of the merge tool work not realizing the upload button was relying on it. I changed the 'value' behavior in the other component to allow bind to work on both spots so everything is happy